circus-train-aws-sns/README.md

```diff
@@ -53,28 +70,110 @@ Currently the S3 credentials specified in the JCEKS key store are used to authen
+|`endTime`|String |Only when `type` is SUCCESS or FAILURE|Time when the Circus Train job ended. Formatted as a date time in ISO8601 format (yyyy-MM-ddTHH:mm:ss.SSSZZ)|
+|`eventId`|String|Always|The Circus Train event id|
+|`sourceCatalog`|String|Only when configured|The configured name of the source catalog|
+|`replicaCatalog`|String|Only when configured|The configured name of the replicat catalog|
+|`sourceTable`|String|Always|The fully qualified name of the source table|
+|`replicaTable`|String|Always|The fully qualified name of the replica table|
+|`replicaTableLocation`|String|Always|The replicate table location|
+|`replicaMetastoreUris`|String|Always|The Hive metastore URIs of the replica Hive Thrift metastore|
+|`partitionKeys`|Ordered Map of String->String|Only on successful replication of a partitioned table|A map where the keys are the partition key names and the values are the partition key types, the order output in the JSON matches the order of the values in the `modifiedPartitions` field below|
+|`modifiedPartitions`|List of List of Strings|Only on successful replication of a partitioned table|A list containing Lists of string representing the partition key values|
+|`bytesReplicated`|Long|Only on replication of `type` SUCCESS of a table containing data|The number of bytes of data replicated|
+|`errorMessage`|String|Only on replication of `type` FAILURE|A message describing the cause of the failure|
+
+### Example Messages
+
+#### Replication start
+The following shows an example JSON message representing the start of a table replication:
+
+	{
+	  "protocolVersion" : "2.0.0",
+	  "type" : "START",
+	  "startTime" : "2016-06-01T15:27:38.365Z",
+	  "eventId" : "ctp-20160601T152738.363Z-CzbZaYfj",
+	  "sourceCatalog" : "sourceCatalogName",
+	  "replicaCatalog" : "replicaCatalogName",
+	  "sourceTable" : "srcDb.srcTable",
+	  "replicaTable" : "replicaDb.replicaTable",
+	  "replicaTableLocation" : "s3://bucket/path",
+	  "replicaMetastoreUris" : "thrift://host:9083",
+	}
+
+#### Replication success for a non-partitioned table
+The following shows an example JSON message representing a successful replication of a non-partitioned table:
+
+	{
+	  "protocolVersion" : "2.0.0",
+	  "type" : "SUCCESS",
+	  "startTime" : "2016-06-01T15:27:38.365Z",
+	  "endTime" : "2016-06-01T15:27:39.000Z",
+	  "eventId" : "ctp-20160601T152738.363Z-CzbZaYfj",
+	  "sourceCatalog" : "sourceCatalogName",
+	  "replicaCatalog" : "replicaCatalogName",
+	  "sourceTable" : "srcDb.srcTable",
+	  "replicaTable" : "replicaDb.replicaTable",
+	  "replicaTableLocation" : "s3://bucket/path",
+	  "replicaMetastoreUris" : "thrift://host:9083",
+	  "bytesReplicated" : 84837488
+	}
+
+#### Replication success for a partitioned table
+The following shows an example JSON message representing a successful replication of a non-partitioned table 
```

Comment:
-> "successful replication of a partitioned table" ?

Reply:
Copy-pasta error, thanks for spotting.